connector/connect/common/src/main/protobuf/spark/connect/commands.proto

```diff
@@ -91,6 +91,16 @@ message WriteOperation {
   // (Optional) A list of configuration options.
   map<string, string> options = 9;
 
+  message SaveTable {
+    string table_name = 1;
+    TableSaveMethod save_method = 2;
+
+    enum TableSaveMethod {
```

Comment:
last thing: according to style guide (which is used in connect protos) https://protobuf.dev/programming-guides/style/#enums , this `enum` start with `_UNSPECIFIED`

```
enum TableSaveMethod {
    TABLE_SAVE_MODE_UNSPECIFIED = 0;

}
```

Reply:
Sure, will address it soon. Thanks!